### ISSUE TITLE
What's the performance of BERT-Large on Squad 2.0 dev ?

### ISSUE THREAD

**Issue by avisil** [google-research/bert#363]
Hi @jacobdevlin-google, could you please share the results of BERT large on Squad 2.0 dev data? And also how many epochs, max length or the hyper-parameter details?

**Comment by eric-haibin-lin** [comment]
Same question here

**Comment by ZhuoranLyu** [comment]
same question, wondering how to achieve the SOTA F1 score on board.

**Comment by SebiSebi** [comment]
Same question. With the params from the README I trained the BERT large to 0.8 F1 score. However, in the SQUAD 2.0 leaderboard, BERT large is reported to have 0.83 F1 score.

**Comment by lxylxyoo** [comment]
Same quesion. I trained a model with `Gradient accumulation` trick and 16GB GPU. The batch_size I used was 24 and the max_seq_length was 384. The F1 score was a little bit higher than 0.8 but lower than 0.81. Hi, @SebiSebi,  could you tell me what's the parameters you used and whether you used GPU or TPU? Thanks!

**Comment by SebiSebi** [comment]
I have trained the model (BERT Large) on a Cloud TPU with the following hyperparameters: 
```
MAX_SEQ_LENGTH = 384
DO_LOWER_CASE = True

LEARNING_RATE = 3e-5
WARMUP_PROPORTION = 0.1

TRAIN_BATCH_SIZE = 24
NUM_TRAIN_EPOCHS = 2.0
SAVE_CHECKPOINTS_STEPS = 1000
ITERATIONS_PER_LOOP = 1000
```

On the SQuAD 2.0 dev set, I got an F1 score of `0.8023`. I tried training the model multiple times but the performance is almost the same accross different runs.

I can see that the score reported on the SQuAD 2.0 leaderboard is referring to a test set which is **not** the same as the dev set. However, I expected the difference to be smaller than 3.0% between test and dev. It would be helpful if someone can share the official set of hyperparameters than can be used to train the model up to 0.83 F1.

**Comment by qianqianhu577** [comment]
Same question here. Has anybody submitted the model with above parameters to the leaderboard to see the test F1?
